bundles/azure/aso/psql/bundle/config/01-flexible-server.ytt.yml

```diff
@@ -0,0 +1,248 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:regexp", "regexp")
+#@ load("@ytt:ip", "ip")
+
+
+#@ flexibleServerName    = data.values.name
+#@ dbSecretName          = data.values.name
+#@ resourceGroupK8SName  = data.values.name
+#@ serviceAccountName      = data.values.name + "-reader"
+#@ serviceAccountNamespace = data.values.namespace
+#@ roleName                = data.values.name + "-reading"
+
+#@ mandatory_fields = [ "name", "namespace", "location", "firewall_rules", "aso_controller_namespace" ]
+#@ missing_fields = []
+#@ for field in mandatory_fields:
+#@   if len(data.values[field]) == 0:
+#@     missing_fields.append(field)
+#@   end
+#@ end
+
+#@ if data.values.database.name == "":
+#@   missing_fields.append("database.name")
```

Comment:
_question_: The other packages default the name of the database to the "server name"'; should we strive for consistency here and do the same?

Reply:
Done.